monetio/models/gocart.py

```diff
@@ -0,0 +1,75 @@
+"""GOCART File Reader."""
+
+import xarray as xr
+
+
+def open_mfdataset(fnames):
```

Comment:
```suggestion
def open_mfdataset(fnames, **kwargs):
```
Can pass `**kwargs` to `xr.open_mfdataset()` (e.g. to drop those variables you didn't want).